plugins/08_BattleForAzeroth/localization/enUS.lua

```diff
@@ -38,6 +38,10 @@ L['dilbert_mcclint_note'] = 'Hey there, name\'s {npc:140461}, Infestation Manage
 L['fizzie_spark_note'] = 'You think your pets have a chance against my Azerite infused team? You wish!'
 L['michael_skarn_note'] = 'Just remember as we start this battle, you asked for this.'
 
+L['cursed_hunter_note'] = 'Kill one of every type of cursed animal to earn the achievement.'
+L['options_icons_cursed_hunter_desc'] = 'Display animal locations for the {achievement:13094} achievement.'
+L['options_icons_cursed_hunter'] = '{achievement:13094}'
```

Comment:
I would like to request that you also add these lines to all the other localizations. But with the change that these are nil.
e.g. `L['cursed_hunter_note'] = nil` 
more infos: https://github.com/zarillion/handynotes-plugins/pull/195#issuecomment-1272416272